pkgs/applications/networking/insync/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "insync-${version}";
+  version = "1.3.13.36129";
+  src = fetchurl {
+    url = "http://s.insynchq.com/builds/insync-portable_${version}_amd64.tar.bz2";
+    sha256 = "18d8ww529nvhwcl5k31qmkzb83k9753ics0dw64w202r8vwbm3cd";
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  postPatch = ''
+    patchelf --interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" client/insync-portable
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -a client $out/client
+    makeWrapper $out/client/insync-portable $out/bin/insync --set LC_TIME C
+  '';
+
+  meta = {
+    platforms = ["x86_64-linux"];
+    license = stdenv.lib.licenses.unfree;
```

Comment:
Maybe it is a good idea to add a comment here saying that it is free for a 15-day trial and is a paid application if you want to continue using it.